### ISSUE TITLE
Tooltip over WYSIWYG buttons

### ISSUE THREAD

**Issue by dashedstripes** [LogicalCMS/Logical#1]
At the moment when you hover over one of the WYSIWYG icons there is no immediate tooltip, yes a tooltip comes up, but only after a while of hovering over.

I suggest adding an instant tooltip on hover, that way the design doesn't get changed but it's much easier for users to know what button they are pressing.

**Comment by dashedstripes** [comment]
I'm happy to look into this if it is desired

**Comment by JoelNichols** [comment]
Ah yes, TinyMCE doesn't do that by default. I'll see about adding this shortly if all goes well.

**Comment by JoelNichols** [comment]
In a nutshell TinyMCE is excellent and implementing this won't be easy. 

Consider this low priority.

**Comment by JoelNichols** [comment]
Did we use TinyMCE then? I genuinely can't remember.

Anyway we're using CKeditor now. And the issue still stands. And it's still not easy to fix.

**Comment by JoelNichols** [comment]
I'm unbelievable slow at fixing issues apparently.

Anyway, will fix soon - looking at other WYSIWYG editors to replace the existing one.
